.release/ci.hcl

```diff
@@ -0,0 +1,210 @@
+schema = "1"
+
+project "vault" {
+  team = "vault"
+  slack {
+    notification_channel = "CRF6FFKEW" // #vault-releases
+  }
+  github {
+    organization = "hashicorp"
+    repository = "vault"
+    release_branches = ["release/1.9.x"]
+  }
+}
+
+event "merge" {
+  // "entrypoint" to use if build is not run automatically
+  // i.e. send "merge" complete signal to orchestrator to trigger build
+}
+
+event "build" {
+  depends = ["merge"]
+  action "build" {
+    organization = "hashicorp"
+    repository = "vault"
+    workflow = "build"
+  }
+}
+
+event "upload-dev" {
+  depends = ["build"]
+  action "upload-dev" {
+    organization = "hashicorp"
+    repository = "crt-workflows-common"
+    workflow = "upload-dev"
+    depends = ["build"]
+  }
+
+  notification {
+    on = "fail"
+  }
+}
+
+event "quality-tests" {
+  depends = ["upload-dev"]
+  action "quality-tests" {
+    organization = "hashicorp"
+    repository = "crt-workflows-common"
+    workflow = "quality-tests"
+  }
+
+  notification {
+    on = "fail"
+  }
+}
+
+event "security-scan" {
+  depends = ["quality-tests"]
+  action "security-scan" {
+    organization = "hashicorp"
+    repository = "crt-workflows-common"
+    workflow = "security-scan"
+  }
+
+  notification {
+    on = "fail"
+  }
+}
+
+event "notarize-darwin-amd64" {
+  depends = ["security-scan"]
+  action "notarize-darwin-amd64" {
+    organization = "hashicorp"
+    repository = "crt-workflows-common"
+    workflow = "notarize-darwin-amd64"
+  }
+
+  notification {
+    on = "fail"
+  }
+}
+
+event "notarize-darwin-arm64" {
+  depends = ["notarize-darwin-amd64"]
```

Comment:
Just curious: why chain these events instead of having them all depend on security-scan?

Reply:
At the moment the orchestrator doesn't support fan out/in, so all the steps need to be in series. Once it's supported, we can update this

Reply:
Makes sense, ty!